src/main/resources/com/google/api/codegen/py/standalone_sample.snip

```diff
@@ -67,7 +67,15 @@
     @# [END {@sample.regionTag}_core]
 
   @# [END {@sample.regionTag}]
-  # FIXME(pongad): add a main method
+
+  from sys import argv
```

Comment:
This is uncommon. The generally-accepted style is `import sys` and then you use `sys.argv` styled as such.